org.jdownloader.JDownloader.metainfo.xml

```diff
@@ -26,7 +26,11 @@
       extrair arquivos automaticamente e muito mais. É um framework fácil de estender que pode
       economizar horas do seu precioso tempo todos os dias!</p>
   </description>
-  <url type="homepage">https://www.jdownloader.org/</url>
+  <url type="homepage">https://jdownloader.org/</url>
+  <url type="contact">https://jdownloader.org/impressum/</url>
+  <url type="faq">https://support.jdownloader.org/knowledgebase/</url>
```

Comment:
```suggestion
  <url type="faq">https://support.jdownloader.org/en/knowledgebase</url>
```

Reply:
Doesn't it make sense to omit the lang component, so people get automatically redirected? E.g. I get redirected to the `/de` site when using a browser that's set to prefer German.

Reply:
Ok, done :)